scout/ecm_prep.py

```diff
@@ -5900,7 +5905,7 @@ def fill_mkts(self, msegs, msegs_cpl, convert_data, tsv_data_init, opts,
                     # Pull TSV scaling fractions and shapes
                     tsv_scale_fracs, tsv_shapes = self.gen_tsv_facts(
                         tsv_data, mskeys, bldg_sect, convert_data, opts,
-                        cost_energy_meas)
+                        cost_energy_meas, save_shp_warn)
```

Comment:
`save_shp_warn` appears to always be an empty list here, why initialize it above and why pass it in this method?